### PR TITLE
Replace the install section in READMEs with link to pub.dev/install

### PR DESCRIPTION
**Pull request by kuhnroyal**
* there is a dedicated install page and this change follows the scheme that official packages are using
* no need to keep the `^replace-with-latest-version` stuff in the docs IMO

### CODE REVIEW

File: dio/README-ZH.md [hunk 1]
```diff
@@ -52,23 +52,9 @@ dio 是一个强大的 HTTP 网络请求库，支持全局配置、Restful API
 
 ## 开始使用
 
-### 添加依赖
```
Comment (AlexV525) [inline dio/README-ZH.md:55]: It's still better than the new one

**Review by AlexV525 — APPROVED** [review]

File: dio/README-ZH.md [hunk 2]
```diff
@@ -52,23 +52,9 @@ dio 是一个强大的 HTTP 网络请求库，支持全局配置、Restful API
...
-你也可以手动将 dio 添加至 pubspec.yaml：
-
-```yaml
-dependencies:
-  dio: ^替换为最新版本
-```
-
-最新稳定版本为：![Pub](https://img.shields.io/pub/v/dio.svg)
-最新包含开发版的版本为：![Pub](https://img.shields.io/pub/v/dio?include_prereleases)
+将 `dio` 包添加到您的 [pubspec 依赖项](https://pub.flutter-io.cn/packages/dio/install).
```
Comment (AlexV525) [inline dio/README-ZH.md:57]:
```suggestion
依照文档将 `dio` 包添加为 [pubspec 的依赖](https://pub.flutter-io.cn/packages/dio/install)。
```

File: dio/README-ZH.md (hunk 1 above)
Comment (kuhnroyal) [reply dio/README-ZH.md:55]: Ok so revert this?

Comment (AlexV525) [reply dio/README-ZH.md:55]: Yes

**Review by AlexV525 — APPROVED** [review]

**Pull request merged**